condensedinlinepanel/edit_handlers.py

```diff
@@ -27,6 +29,8 @@ def default(self, o):
             return o.strftime(fmt)
         elif isinstance(o, datetime.time):
             return o.strftime('%H:%M')
+        elif isinstance(o, StreamValue):
```

Comment:
Is this solving a different issue?

Reply:
Ah, yes it is actually, I was using the panel with streamfield somewhere so it was barfing on that as well. I needed to do this to get the site up as I was running it during testing.

Reply:
Thanks, I'll double check that as well and add another release note!

Reply:
Great stuff, many thanks for all the effort!

Reply:
Streamfields seem to have an issue with saving, so I won't mention this one yet, but no harm in having this in now though! Thanks!